package.json

```diff
@@ -190,7 +192,6 @@
     "react-router-dom": "^4.2.2",
     "react-select": "^1.0.0",
     "react-simple-file-input": "^2.0.0",
-    "react-tabs": "^3.1.0",
```

Comment:
Why are we removing this dependency?

Reply:
I have explained that on the commit comment, we have our own tabs component now, this one was incompatible.